korean/cry/cry-more.md

```diff
@@ -0,0 +1 @@
+울음 소리를 듣고 왔는지 갑자기 나타난 다람쥐! 눈이 마주치자 달아납니다. 어떻게 할까요?
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.